guides/prow_setup.md

```diff
@@ -1,12 +1,42 @@
 # Prow setup
 
-## Adding new Kubernetes resources
+All Prow related config files for [prow.knative.dev](https://prow.knative.dev)
+are under [config/prow](../config/prow). There is also a staging Prow in
+[prow-staging.knative.dev](https://prow-staging.knative.dev) for testing the new
+config changes before rolling out to production, and all its config files are
+under [config/prow-staging](../config/prow-staging).
 
-1. Check under [prow/cluster](../config/prow/cluster) and add a new file there
-   if not exist.
+## Modify Prow cluster configs
+
+Prow is a collection of microservices that are deployed on a Kubernetes cluster,
+and all the Kubernetes object YAML files (except secrets) are under
```

Comment:
Maybe `Kubernetes resources definitions`?

Reply:
I feel "Kubernetes object `.yaml` files" is more accurate. In https://kubernetes.io/docs/concepts/overview/working-with-objects/kubernetes-objects/, I found  something like `In the .yaml file for the Kubernetes object you want to create`.

When you search for `Kubernetes resources definitions` , most of the results are about `Kubernetes Custom Resource Definitions`, and it seems normally we do not use this acronym for Kubernetes native resources.

Reply:
Ah. I see, these are actually k8s services not resources